dashboard/src/components/SchemaForm/FieldTemplate.tsx

```diff
@@ -3,10 +3,15 @@ import { FieldTemplateProps } from "react-jsonschema-form";
 
 // adapted from https://jsfiddle.net/hdp1kgn6/1/
 const FieldTemplate: React.SFC<FieldTemplateProps> = props => {
-  const { id, classNames, label, children, displayLabel } = props;
+  const { id, classNames, label, children, displayLabel, required } = props;
   return (
     <div className={classNames}>
-      {displayLabel && <label htmlFor={id}>{label}</label>}
+      {displayLabel && (
+        <label htmlFor={id}>
+          {label}
+          {required && " *"}
```

Comment:
is quite obvious but a line at the end saying `* Mandatory fields` would be nice

Reply:
What about changing it to `(required)`? I want to avoid adding an extra thing at the end of the form, the markup is already fairly messy.

If you're worried about users simply missing the required fields, the `required` html attribute is set so browsers should catch the error before forms are submitted.

Reply:
👍 that's okay